.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file is quite specific to your setup/filesystem. Not sure it needs to be checked in unless you can make it generic.

Reply:
I'll add .vscode to .gitignore ok?